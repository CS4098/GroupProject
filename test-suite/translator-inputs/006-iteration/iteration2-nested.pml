process iteNest {

    iteration {
        action a {
            provides { c }
        }
        iteration {
            action c {
                requires { c }
                provides { d }
            }
            action d {
                requires { d }
                provides { b }
            }
        }
        action b {
            requires { b }
        }
    }
}
