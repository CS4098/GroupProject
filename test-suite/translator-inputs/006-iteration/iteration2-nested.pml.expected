bool b;
bool c;
bool d;

active proctype iteNest()
{
	int count__;
    for (count__ : 1 .. 4) {
        c = true;
        int count___;
        for (count___ : 1 .. 4) {
            c ->
                d = true;
            d ->
                b = true;
        }
        b ->
    }
}
